origin-growth/src/apollo/resolvers.js

```diff
@@ -59,14 +62,19 @@ const resolvers = {
     },
     async campaign(root, args, context) {
       requireEnrolledUser(context)
+      const walletAddress = (await getUser(context.authToken)).ethAddress
```

Comment:
suggestion: It would be nice if ethAddress would be provided in the context ? This way no need to call getUser in every resolver.

Reply:
good suggestion tnx